src/chrome/content/rules/dellin.ru.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Delovie Linii (transport company)">
+	<target host="dellin.ru" />
+	<target host="*.dellin.ru" />
+	<securecookie host="(.+\.)?dellin\.ru$" name=".+" />
+	<rule from="^http://(.+\.)?dellin\.ru/" to="https://www.dellin.ru/" />
```

Comment:
failure: dellin.ru.xml failed test: The 'from' rule contains unescaped period in regular expression.  Try escaping it with a backslash.



I guess it's supposed to be there, but I guess it doesn't like the .+ syntax